source/administration/config-settings.rst

```diff
@@ -4037,6 +4037,102 @@ Changes to this setting require a server restart before taking effect.
 Experimental Settings only in config.json
 -----------------------------------------
 
+Audit settings
+~~~~~~~~~~~~~~~
+The audit settings output audit records to syslog (local or remote server via TLS) and/or to a local file. Both are disabled by default.
+
+Syslog configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: When ``true`` syslog output is enabled.
+**False**: Syslog output is disabled.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogEnabled": []`` with options ``true`` and ``false``.                                   |
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Syslog IP
+^^^^^^^^^^^^^^
+The IP address or domain of the syslog server. Use ``localhost`` for local syslog.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogIP": []`` with string array input consisting of an IP address or domain name.            |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Syslog port
+^^^^^^^^^^^^^^
+The port that the syslog server is listening on.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogPort": []`` with string array input consisting of a port number.            |
```

Comment:
@wiggin77, is there a default port that should be used?

Reply:
The default is 6514.   It is a numeric (not string).   Also, should the JSON 
```json
"ExperimentalAuditSettings": {
    "SysLogPort": 6514,
}
```
become `ExperimentalAuditSettingsSysLogPort` with no separator?  I've not worked with RST before so I don't know how that gets reformatted.

Reply:
I'll review and revert on that.

Reply:
AFAIK there is not meant to be a separator (based on existing content), but I am tagging @amyblais to confirm.

Reply:
Reviewing the document, `ExperimentalAuditSettings` is the only section where the section name and key are joined like that.  As a user reading the doc, I would think `ExperimentalAuditSettingsSysLogPort` is a top-level key, but that wouldn't work.

Reply:
Thanks, @wiggin77. I did a bit more digging into this and I based the format on the existing experimental config settings that are prefixed by "Experimental". I've updated the docs but was wondering if it's something that would be considered in a future release (appending "ExperimentalAuditSettings" to the setting)?

Reply:
From what I see in config.json, when there is a single experimental option within an existing feature or section then the key is prefixed with `experimental`, such as `ExperimentalGroupUnreadChannels` within `ServiceSettings`.   When a larger feature is experimental it gets it's own section and the section is prefixed `experimental`.

Reply:
Ah! Yeah that makes sense, thanks for looking into it. I've made the changes - thank you!